lib/sensors/accelerometer.cpp

```diff
@@ -0,0 +1,159 @@
+#include "accelerometer.hpp"
+
+namespace hyped::sensors {
+
+Accelerometer::Accelerometer(const std::uint8_t kDeviceAddr,
+                             const std::uint8_t channel,
+                             io::HardwareI2c &i2c,
+                             core::ILogger &log)
+    : kDeviceAddr(kDeviceAddr),
```

Comment:
also this feels weird, defining a constant in the constructor. idk if it goes against the style guide but i haven't seen it elsewhere